packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -15,7 +15,7 @@
 //! You can also provide a fallback function, which is called whenever a hook hasn't been added for
 //! a specific type of attachment.
 //! The fallback function needs to have a signature of
-//! `Fn(&Frame, &mut HookContext<T>) -> Option<Emit>`
+//! `Fn(&Frame, &mut HookContext<T>) -> Diagnostics`
```

Comment:
incorrect docs, should be `Vec<Emit>`

Reply:
fixed in [4892398](https://github.com/hashintel/hash/pull/976/commits/4892398d338cb681fe7bbd5443b584fdc6383fad)